### ISSUE TITLE
Markdown hyperlink formatting appears to be backwards

### ISSUE THREAD

**Issue by Zhetadelta** [enchanted-sword/ch-utils#27]
Zhere is a disconnect between za live-preview feature of chutils and za cohost-provided preview when dealing wizh markdown links as demonstrated in za screenshots

**Screenshots**
![image](https://github.com/user-attachments/assets/5105e365-9218-4372-b66a-9561e3dd679c)
![image](https://github.com/user-attachments/assets/64e5de19-ba72-4e3f-8037-ec1bb8f939fa)

on Firefox 129.0.2

**Comment by enchanted-sword** [comment]
Whoops, looks like we accidentally put a second instance of `href` in the link body instead of `text`.
```js
const link = ({ text, href }) => text === href ? `
  <div class="co-embed">
    <div class="renderIfVisible">
      <div>
        <div>
          <div
            style="left: 0px; width: 100%; height: 160px; position: relative; border-radius: 3px; box-shadow: rgba(0, 0, 0, 0.06) 0px 8px 18px; overflow: hidden; padding-bottom: 0px;">
            <iframe
              src="https://iframely.net/api/iframe?app=1&amp;url=${encodeURIComponent(href)}&amp;key=${IFRAMELY_KEY}"
              style="top: 0; left: 0; width: 100%; height: 100%; position: absolute; border: 0;" allowfullscreen=""
              tabindex="0"></iframe></div>
        </div>
      </div>
    </div>
    <div class="co-ui-text mt-0 p-3 text-right"><a href="${href}" target="_blank" rel="noopener nofollow" tabindex="0">${text}</a></div>
  </div>
` : `<a href="${href}">${href}</a>`;
```

**Comment by enchanted-sword** [comment]
fixed in a37c5c49e64879a9834706245619c96209418d6e
